hook/suite_test.go

```diff
@@ -42,6 +42,7 @@ const (
 	topolvmProvisioner3StorageClassName         = "topolvm-provisioner3"
 	topolvmProvisionerImmediateStorageClassName = "topolvm-provisioner-immediate"
 	hostLocalStorageClassName                   = "host-local"
+	unexistStorageClassName                     = "doesnot-exists"
```

Comment:
`doesnot-exists` looks strange.  The same applies to `unexist`.

```suggestion
	missingStorageClassName                     = "missing-storageclass"
```